cpp/util/etcd.h

```diff
@@ -145,9 +147,14 @@ class EtcdClient {
   void WatchRequestDone(WatchState* state, GetResponse* gen_resp,
                         util::Task* child_task);
 
+  void MaybeLogEtcdVersion();
+
+  util::Executor* const executor_;
+  std::unique_ptr<util::SyncTask> log_version_task_;
```

Comment:
Why have this in a `unique_ptr`, instead of by value? It's created at startup, and lasts the whole time...

Incidentally, maybe add a comment to `~EtcdClient` to mention it could block.